Python/compile.c

```diff
@@ -7157,11 +7156,6 @@ instructions_to_instr_sequence(PyObject *instructions, instr_sequence *seq)
             goto error;
         }
     }
-    if (seq->s_used && !IS_TERMINATOR_OPCODE(seq->s_instrs[seq->s_used-1].i_opcode)) {
-        if (instr_sequence_addop(seq, RETURN_VALUE, 0, NO_LOCATION) < 0) {
-            goto error;
-        }
-    }
```

Comment:
I guess we have no unittests depending on this automatic insertion of a `RETURN_VALUE`?

Reply:
We don't, and also this was wrong because it was missing the LOAD_NONE (or it should be RETURN_NONE).

We can add that if we need, but I think it's nice if the test code returns something that can be checked for correctness.